pypeman/commands.py

```diff
@@ -67,26 +67,8 @@ def sig_handler_func(loop, signal, ctx):
     loop.create_task(sig_handler_coro(loop, signal, ctx))
 
 
-def load_project():
-    settings.init_settings()
-    project_module = settings.PROJECT_MODULE
-    try:
-        importlib.import_module(project_module)
-    except ImportError as exc:
-        msg = str(exc)
-        if 'No module' not in msg:
-            print("IMPORT ERROR %s" % project_module)
-            raise
-        if project_module not in msg:
-            print("IMPORT ERROR %s" % project_module)
-            raise
-        print("Missing '%s' module !" % project_module)
-        sys.exit(-1)
-    except Exception:
-        traceback.print_exc()
-        raise
-
-
+# TODO: might rename main and refactor into pypman.graph
+# could call it perhaps. run_graph() or load_and_run_graph()
```

Comment:
If my memory is good, the main is used for stating pipeline and not just to draw the graph ?

Reply:
`main()` handles some flags, loads the graph, adds sigal handlers, and start channels, endpoints, cli (REPL), debug interface and  the graph and destroys it after a stop request.

I think the name `load_and_run_graph()` might be a better name, than `main()`

Reply:
Further clarification. (Perhaps not needed, but just in case):

The word `graph` in `pypeman.graph` is intended for anything that works with the graph (graph being the set of all channels and the nodes and the connections between the nodes.

so functions to load the graph, to get a graphical representation, to start it to do anything.

In my opinion `pypeman.commands` should just contain a front end to other modules and deal as far as possible only with the CLI part, but as little real functionality as possible

Reply:
Thanks for the clarification. I'm Ok then !